base/docs/basedocs.jl

```diff
@@ -644,6 +644,31 @@ desired can be used.
 """
 kw"if", kw"elseif", kw"else"
 
+"""
+    a ? b : c
+
+Short form for conditionals; read "if `a`, evaluate `b` otherwise evaluate `c`".
+Also known as the
```

Comment:
```suggestion
Also known as the [ternary operator](https://en.wikipedia.org/wiki/%3F:).
```

Reply:
The Julia docs do have a julia-specific discussion of the ternary so I think it's better to link to that. (Only problem is that it's not in its own section, so cannot be directly linked to.)

Reply:
Ah I see you've linked it further down. Seems reasonable, let me check whether it works.

Reply:
Doesn't work :-(